### ISSUE TITLE
Not Receiving 34.6.2 Update via Chrome Web Store

### ISSUE THREAD

**Issue by CyclonicPinna** [KC3Kai/KC3Kai#2997]
Hi there. I'm running KC3 Kai on my old Windows XP SP3 PC (32 bit). Chrome version is 49.0.2623.112.

The current update version I received via Chrome Web Store was 34.6.1. I can already see in the Web Store page that the latest version is 34.6.2, but several days have passed and I have still not received the update.

I think KC3 Kai is still compatible with the old Chrome version, so I'm not sure if it is an issue from the Web Store side. I remember there was also a similar issue some time ago when new updates could not be received via Chrome Web Store, but at that time the Web Store page would not show newer versions, which is different from the situation this time.

Thank you for your efforts.

**Comment by sinsinpub** [comment]
![image](https://user-images.githubusercontent.com/160240/83109112-6037fb00-a0f3-11ea-9831-30ffdf72c12d.png)

did u try the 'Update extensions now' button?
if not work, should be webstore side or your internet connection issue 🤔

**Comment by CyclonicPinna** [comment]
Yes, I have tried it several times during these days but no update was installed. I have no connection issue with the Web Store so I believe Internet is not the cause.

**Comment by CyclonicPinna** [comment]
Hi there. I have tried some other steps and see where the problem lies. After deleting / removing KC3 Kai an trying to reinstall it via the Web Store, it just shows an error message "Download Interrupted". This should be the reason why I can see a newer version in the Web Store page but receive no update. The quick solutions online for this issue do not work for me, so I guess I will try manually installing the extension via Github.

**Comment by temacort** [comment]
Same here from Windows 10 version 1511 and Chrome 54.0.2840.71 (64 bit)
